### PR TITLE
[FEATURE] Add account_set_domain functionality

### PR DESCRIPTION
**Pull request by wilsonianb**
DEC-464
+r

### CODE REVIEW

File: lib/account_set_domain.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+var fs = require('fs')
+var ripple = require('ripple-lib')
+var Wallet = require('ripple-wallet-generator')({
```
Comment (stevenzeiler) [inline lib/account_set_domain.js]: `ripple-lib` already supports the ripple wallet generator functionality.

Comment (stevenzeiler) [inline lib/account_set_domain.js]: `require('ripple-lib').Wallet.generate()`

Comment (stevenzeiler) [inline lib/account_set_domain.js]: https://github.com/ripple/ripple-lib/blob/develop/docs/GUIDES.md#generating-a-new-ripple-wallet

Comment (wilsonianb) [inline lib/account_set_domain.js]: Ok, `ripple-wallet-generator` has been removed

**Comment by stevenzeiler** [comment]
:+1: LGTM

**Pull request merged**